servicenow/icon_servicenow/triggers/incident_created/trigger.py

```diff
@@ -0,0 +1,68 @@
+import komand
+import time
+from .schema import IncidentCreatedInput, IncidentCreatedOutput, Input, Output, Component
+# Custom imports below
+
+
+class IncidentCreated(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='incident_created',
+                description=Component.DESCRIPTION,
+                input=IncidentCreatedInput(),
+                output=IncidentCreatedOutput())
+        self.found = {}
+        self.url = ""
+        self.method = "get"
+        self.query = ""
+
+    def initialize(self):
+        if self.query:
+            response = self.connection.request.make_request(
+                self.url, self.method, params=self.query)
+        else:
+            response = self.connection.request.make_request(self.url, self.method)
+
+        try:
+            results = response.get("resource").get("result")
+        except KeyError as e:
+            raise PluginException(preset=PluginException.Preset.UNKNOWN,
+                                  data=response.text) from e
+
+        sys_ids = [result.get("sys_id") for result in results]
+        for sys_id in sys_ids:
+            self.found[sys_id] = True
```

Comment:
```suggestion
            self.found.add(sys_id)
```
`self.found` should be a set - the `sys_id` key can never be False, therefore there's no need to store a boolean there. Just the presence of the sys_id in the set should indicate that it has been seen already.